pkg/operator/sync.go

```diff
@@ -126,6 +144,70 @@ func (optr *Operator) syncTerminationHandler(config *OperatorConfig) error {
 	return nil
 }
 
+func (optr *Operator) syncWebhookConfiguration(config *OperatorConfig) error {
+	if err := optr.syncValidatingWebhook(
+		optr.kubeClient.AdmissionregistrationV1().ValidatingWebhookConfigurations(),
+		mapiv1.NewValidatingWebhookConfiguration(),
```

Comment:
why does this need to be parameters for syncValidatingWebhook?
can we have a unit test for syncWebhookConfiguration?